### ISSUE TITLE
KodaDot V3 Redesign + Guidelines 

### ISSUE THREAD

**Issue by XyloDrone** [kodadot/nft-gallery#1619]
### Is your feature request related to a problem?

Hello everyone, let’s escape the Craigslist pit and turn KodaDot into a slick NFT Explorer. 👯  
If you need me to respond, I am usually available Mon-Fri, and at my computer from 9am-6pm CET, whilst working 3x days per week. 

Buckle up because this is a long document which aims to: 

- Organise all issues in a large [mind map](https://miro.com/welcomeonboard/bmQwQUkzUWNJWXpzM1lzQVprZUtncFdtZ1BzSlpXRjNkSGFoWjN4RWZzS3I2SXl5N09KTklqT2ZhN21aNkRmY3wzNDU4NzY0NTEzOTE4ODYxNDIy?invite_link_id=846027370969) which will serve as a guide on where to place each PR/issue 

- eliminate double work, outdated PRs/issues and double PRs/issues 

- construct a way for all of us to easily communicate with each other and move rapidly forward

I have started redesigning page by page, so far I’ve finished the “Nav Bar”, “Gallery”, “Jobs” and “Checkout” designs. I would suggest either starting with the “Gallery” or waiting for me to finish further designs. 

Please tag me in any new PRs/issues being opened that have any relation to UX/UI so I can keep adding these to the mind map accordingly + providing designs so that devs don’t have to bother with this, as it is my duty to sort out UX/UI so you can focus on what you do best. 

Redesign file is [here](https://www.figma.com/file/GeVzZKVORd9L8kAWrlKtZv/KodaDot-UX%2FUI?node-id=0%3A1)

**Further Notes:** 
I would suggest holding off on any tutorials and FAQ before the redesign is complete, however, I'd assign @cryptodamsky and @onerawartist to do these once the redesign is done. In order to fast track this process I will focus on redesigning the landing page and minting user journeys next. 

**[Mind Map Legend](https://miro.com/welcomeonboard/bmQwQUkzUWNJWXpzM1lzQVprZUtncFdtZ1BzSlpXRjNkSGFoWjN4RWZzS3I2SXl5N09KTklqT2ZhN21aNkRmY3wzNDU4NzY0NTEzOTE4ODYxNDIy?invite_link_id=342226923669)**

Cyan color: Pages 
Black: Unresolved issues with design 
Orange: Resolved issues with design 
Note: some blocks are duplicated due to having a pressence on multiple pages 

All of your suggestions are very much appreciated, let me know how you'd like to proceed and please assign yourselves to issues/PRs you see fit. 

### Describe the solution you would like

Note: As I continue to research user journeys, some things are subject to change, I will update as I go. 

**General changes:** 

- [ ] - Change color to E6007E
- [ ] - Update background color to: 0A0A0A
- [ ] - remove glow from images
- [ ] - remove zoom in from images

The design will sort out the issues/PR's per page, as in the list below: 

**Nav Bar**
#910 
#1337

My proposal is to remove these from nav bar: 
**"Collections"**(these should be easy to view in gallery, not needed in NAV at all) 
**"Spotlight"** (I think this should be merged with "series" as a separate tab and I suggest renaming "series" to "leaderboad", as the term is confusing) 
I suggest adding **"credit", "transform", "admin", "settings"** under a new page called **"My profile"** where the user will be able to edit and access multiple features; see the works/artists/collections they like/follow, what they have collected, active data on the works they purchased/created. Additionally, the page would have a "dev" dashboard where we can also track all of our Github contributions and activities(have a look at the [mind map](https://miro.com/welcomeonboard/bmQwQUkzUWNJWXpzM1lzQVprZUtncFdtZ1BzSlpXRjNkSGFoWjN4RWZzS3I2SXl5N09KTklqT2ZhN21aNkRmY3wzNDU4NzY0NTEzOTE4ODYxNDIy?invite_link_id=445095647518) for a breakdown of the page). Thsi page will take a while to researh but I am confident we can have the redesign of the entire platform done within the next 2 weeks. 

**Personal account zone: My Profile**

- [ ] #911 
- [ ] #1432 
- [ ] #1526 
- [ ] #1516 once #744 adjusted 
- [ ] #1334 
- [x] #1172 
- [x] #1285 
- [x] #942 
- [x] #1439 
- [x] #361 

**Tutorial** would be turned into "Help" and it would consist of both tutorials + FAQ's 

**Landing**

- [ ] #90 
- [ ] #1115 
- [ ] #1489 
- [ ] #1445 
- [ ] #1564 
- [x] #876 
- [x] #899 
- [x] #145 
- [x] #145 
- [x] #319 
- [x] #660 
- [x] #837 
- [x] #899 

*Activity(a new page in the navigation to track activity in real time)* 

- [ ] #1236 

**Leaderboard/Series**
Add as tabs:

- [ ] #1440 
- [x] #1477 

**Gallery+Search Engine**

- [ ] #348 
- [ ] #859 
- [ ] #290
- [ ] #903 
- [ ] #285
- [ ] #1023 
- [ ] #1088 
- [ ] #1191 
- [ ] #1526 
- [ ] #1174 
- [ ] #991 
- [x] #49 
- [x] #1050 
- [x] #802 
- [x] #35 
- [x] #1367 
- [x] #42
- [x] #1294 
- [x] #979 
- [x] #1367 
- [x] #1411 
- [x] #979 
- [x] #1582 
- [x] #1231 

**Create(Minting)**

- [ ] #378 
- [ ] #460
- [ ] #419 
- [ ] #614 
- [ ] #753 
- [ ] #847 
- [ ] #614 
- [ ] #911 
- [ ] #991 
- [ ] #1005 
- [ ] #1094 
- [ ] #1330 
- [ ] #1406 
- [ ] #1526 
- [ ] #991 
- [x] #575 
- [x] #1015 
- [x] #401 

**Profile(this is what others view when they ladn on your profile, the "My profile" section is private)**

- [ ] #617
- [ ] #1190 
- [ ] #880 
- [ ] #825 
- [ ] #898 
- [ ] #903 
- [ ] #938 
- [ ] #716 
- [x] #612 
- [x] #678 
- [x] #744 
- [x] #571 
- [x] #49 
- [x] #877 
- [x] #1015 
- [x] #1285 
- [x] #1029 

**NFT + Collection view(I need to separate these accordingly as issues are bundled together at the moment)**

- [ ] #219 
- [ ] #357
- [ ] #901 
- [ ] #1330 
- [ ] #910 
- [ ] #1083 
- [ ] #1214 
- [ ] #1606 
- [ ] #936 
- [ ] #1252 
- [ ] #14 
- [ ] #1010 
- [ ] #1191 
- [ ] #1174 
- [ ] #878 
- [ ] #1323 
- [ ] #1423 
- [ ] #1012 
- [ ] #1483 
- [ ] #1115 
- [ ] #1122 
- [ ] #991 
- [ ] #1566 
- [ ] #348 
- [x] #1019 
- [x] #958 
- [x] #594
- [x] #1118
- [x] #957 
- [x] #612 
- [x] #744 
- [x] #1238 
- [x] #737 
- [x] #1160 
- [x] #962 
- [x] #1015 
- [x] #1175 
- [x] #1065 
- [x] #1029 
- [x] #979 
- [x] #1456 
- [x] #1111 
- [x] #1390 

**Leaderboard(previously Series)**

- [ ] #827 
- [ ] #1165 
- [ ] #1165
- [ ] #1496 
- [x] #1386 
- [x] #49 

**Help: FAQ+Tutorials**

- [ ] #847 
- [ ] #614 
- [ ] #725 
- [ ] #915 
- [x] #1015 
- [x] #1604 
- [x] #1015 
- [x] #648
- [x] #737 
- [x] #1220 
- [x] #529 
- [x] #648 
- [x] #401 

**Footer**
Quite a few things added in the design, but only 1 linking to issues, which is #702 : jobs 

### Describe alternatives you have considered

I suppose there will be plenty once we start unpacking page by page, these will be added here once we encounter obstacles. 

### Screeenshots

Designs are on this link, if you have any questions or require exports, just respond to this thread and tag me. 
https://www.figma.com/file/GeVzZKVORd9L8kAWrlKtZv/KodaDot-UX%2FUI?node-id=0%3A1

**Comment by roiLeo** [comment]
Hey!
Dope work, love the new design!

Few things:
- I have a concern about accessibility color contrast: text => #E6007E on background => #1F1F1F
- can we have a light theme page example? 
- is "checkout flow" possible feature wise?
- I find search filters hard for the UX, would it be possible to rework a version less harmful? maybe create a sidebar filter
- Not a fan of gradiant button. What would be hover/active state?

What are your thoughts on this?

**Comment by XyloDrone** [comment]
> Hey! Dope work, love the new design!

Thank you! 



> Few things:
> 
>     * I have a concern about accessibility color contrast: text => #E6007E on background => #1F1F1F

I've changed the pink color to #FF47AC and bacground to #090909, see new design [here](https://www.figma.com/file/GeVzZKVORd9L8kAWrlKtZv/KodaDot-UX%2FUI?node-id=34%3A94). Ran it through a contrast checker plug in which approved it :D 

>     * can we have a light theme page example?

[Here](https://www.figma.com/file/GeVzZKVORd9L8kAWrlKtZv/?node-id=128%3A1257) we go

>     * is "checkout flow" possible feature wise?

I suppose this is a Q for @yangwao, but the suggestions was mentioned in #910 

>     * I find search filters hard for the UX, would it be possible to rework a version less harmful? maybe create a sidebar filter
>     * Not a fan of gradiant button. What would be hover/active state?
Any particular reason why you dislike the gradient? I've positioned the filters at the top so that the images have more space, [here's](https://www.figma.com/file/GeVzZKVORd9L8kAWrlKtZv/?node-id=0%3A1) an example with the sidebar and solid buttons.  

In regards to the UX of filtera, @cryptodamsky just submitted a proposal to add info to certain sections in #1637 , I think this would be a great addition if we agree on it.

**Comment by yangwao** [comment]
![](https://media.giphy.com/media/xTiTnyIbhHnVNyjm6c/giphy.gif) 

I have no words for this amazing issue and effort someone goes through all of the issues and trying index them into one and whole kodadot v3! A really impressive amount of effort and thank you in the first place!

> eliminate double work, outdated PRs/issues and double PRs/issues

if there are, it's bc they left of old process and some of might be reinforcement by logging twice which is good, also be aware that some issues sound similar, they are not. Definitely would chop on our backlog, but some older issues are gems which just weren't implemented fully :)

> construct a way for all of us to easily communicate with each other and move rapidly forward

github made discussion boards, where information doesn't lost as in realtime chats
![image](https://user-images.githubusercontent.com/5887929/147881255-dd65a73a-92fa-44d4-a2bd-49cebe1e750c.png)

> I would suggest holding off on any tutorials and FAQ before the redesign is complete

Rather have tutorial and onboarding material, because migration to new beta took us over 3 months tbh and can't afford loose that amount of time. Rather have outdated onboarding material and as separate process we can upgrade them and learn by doing new one.

> "Collections"(these should be easy to view in gallery, not needed in NAV at all)

yes this should be merged as functionality in the gallery I guess, as collection could be enabled there, but if collection is interesting, would be nice users have dedicated entrance point to that

> "Spotlight" (I think this should be merged with "series" as a separate tab and I suggest renaming "series" to "leaderboad", as the term is confusing)

Yes and no. Leaderboard is actually term I'm trying steeraway as it put artist into competition bit which is not aim of Spotlight. Rather would be good name to have Spotlight renamed to Artist Directory.

> I suggest adding "credit", "transform", "admin", "settings" under a new page called "My profile"

yeah would be nice to have something like "Control Center" I guess

Re design in firma, I would strongly require to have this time some sort of atomic components, before we go for v3
Seems we had tons of issue double fixed or missing somewhere which introduced lot of incosistency across codebase and wasted pretty lot amount of time adjusting components per each instance.

Re design process I would advise starting it as mobile-first design, this seems way much as desktop only so till we will have decide on mobile it can take time. 

Plus I don't see anywhere AR/VR/XR buttons tho for our XR efforst

**Comment by XyloDrone** [comment]
> ![](https://camo.githubusercontent.com/66ad6b256ae15d60f537ee40397e6f573b6056e40de72e82f5b007cbc9528761/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f785469546e79496268486e564e796a6d36632f67697068792e676966)
> 
> I have no words for this amazing issue and effort someone goes through all of the issues and trying index them into one and whole kodadot v3! A really impressive amount of effort and thank you in the first place!

Thank you! Hopefully it will help us build stuff faster! 

> > eliminate double work, outdated PRs/issues and double PRs/issues
> 
> if there are, it's bc they left of old process and some of might be reinforcement by logging twice which is good, also be aware that some issues sound similar, they are not. Definitely would chop on our backlog, but some older issues are gems which just weren't implemented fully :)

Fair enough and understandable. 

> > construct a way for all of us to easily communicate with each other and move rapidly forward
> 

> > I would suggest holding off on any tutorials and FAQ before the redesign is complete

> Rather have tutorial and onboarding material, because migration to new beta took us over 3 months tbh and can't afford loose that amount of time. Rather have outdated onboarding material and as separate process we can upgrade them and learn by doing new one.

Alright, I believe @onerawartist made a tutorial for minting, so that can be added and @cryptodamsky said he is also willing to make some materials. I've updated the landing page design and created a dedicated section for newcomers. I'll keep working on the entire user journey for new useers and we can add the videos etc there. 

> > "Collections"(these should be easy to view in gallery, not needed in NAV at all)
> 
> yes this should be merged as functionality in the gallery I guess, as collection could be enabled there, but if collection is interesting, would be nice users have dedicated entrance point to that

It could also be in a drop down from "Gallery?" 

> > "Spotlight" (I think this should be merged with "series" as a separate tab and I suggest renaming "series" to "leaderboad", as the term is confusing)

> Yes and no. Leaderboard is actually term I'm trying steeraway as it put artist into competition bit which is not aim of Spotlight. Rather would be good name to have Spotlight renamed to Artist Directory.

Understandable, but I do believe that any kind of table suggests competition, regardless, the term was suggested primarily for easier UX.  

> > I suggest adding "credit", "transform", "admin", "settings" under a new page called "My profile"

> yeah would be nice to have something like "Control Center" I guess

Cool! 
 
> Re design in firma, I would strongly require to have this time some sort of atomic components, before we go for v3 Seems we had tons of issue double fixed or missing somewhere which introduced lot of incosistency across codebase and wasted pretty lot amount of time adjusting components per each instance.

Fair enough. 

> Re design process I would advise starting it as mobile-first design, this seems way much as desktop only so till we will have decide on mobile it can take time.

I'll start working on a mobile version of already designed pages so we can have an idea of how it looks like. 

> Plus I don't see anywhere AR/VR/XR buttons tho for our XR efforst
This will be on the artist profiles and NFTs themselves+ in the filters for search. 
If we have some pre-built galleries we can add this to the Nav Bar and/or emphasise in landing.
